LICENSE-binary

```diff
@@ -404,8 +404,7 @@ org.apache.flink:flink-shaded-netty:4.1.49.Final-12.0
 org.apache.flink:flink-shaded-zookeeper-3:3.4.14-12.0
 org.apache.flink:flink-streaming-java_2.12:1.12.0
 org.apache.flink:force-shading:1.12.0
-org.apache.helix:helix-common:1.0.4
```

Comment:
helix-common is not used, so removed it.

Reply:
Are you sure? helix-common is pulled by helix-core. You are missing the following 3 lines as well

Reply:
I thought only direct dependencies are listed in the LICENSE file. Brought back the helix-common and fixed license in the rest of helix libs.